mando/core.py

```diff
@@ -30,6 +30,25 @@ def __init__(self, prog=None, version=None, **kwargs):
         self.subparsers = self.parser.add_subparsers()
         self.argspecs = {}
         self.current_command = None
+        self.options = None
+
+    @property
+    def name(self):
+        return self.parser.prog
+
+    # Add global script options.
+    def option(self, *args, **kwd):
+        assert args and all(arg.startswith('-') for arg in args), \
+            "Positional arguments not supported here"
+        completer = kwd.pop('completer', None)
+        arg = self.parser.add_argument(*args, **kwd)
+        if completer is not None:
+            arg.completer = completer
+        return arg
+
+    # Attribute lookup fallback redirecting to (internal) options instance.
+    def __getattr__(self, attr):
+        return getattr(self.options, attr)
```

Comment:
That's a convenience accessor to get the global options. Supports something like that:
```
Script = mando.core.Program()
Script.option("--foo", dest='bar')
```
And in some of the subcommands (after actual argument parsing):
```
print(Script.bar)
```
Note the different var name (if not deduced automatically by ArgumentParser()).
Besides that it's not used internally, and hence, might not be included in the tests yet.
Please also note the comment in my last e-mail about attribute names.